build/Dockerfile

```diff
@@ -16,29 +16,25 @@ ENV PATH=${PATH}:/usr/local/go/bin
 
 # Build WMCB
 WORKDIR /build/
-RUN git clone --single-branch --branch release-4.6 https://github.com/openshift/windows-machine-config-bootstrapper.git
-WORKDIR windows-machine-config-bootstrapper
+COPY windows-machine-config-bootstrapper .
 RUN make build
 
 # Build hybrid-overlay
 WORKDIR /build/
-RUN git clone --single-branch --branch release-4.6 https://github.com/openshift/ovn-kubernetes/
-WORKDIR ovn-kubernetes/go-controller/
+COPY ovn-kubernetes .
+WORKDIR go-controller/
```

Comment:
Shouldnt this be WORKDIR `ovn-kubernetes/go-controller` ?

Reply:
How did `operator-sdk build` work with `go-controller`?

Reply:
@aravindhp here I wasn't creating directories inside build, hence COPY ovn-kubernetes . copies all contents inside of ovn-kuberenetes to /build, since the WORKDIR prior to this step was /build/ it was working fine with WORKDIR being set to /go-controller. Pushing changes now to create those directories and then copy the contents.